lib/stat_tracker.rb

```diff
@@ -218,4 +218,33 @@ def lowest_scoring_home_team
     worst_home_scorer["teamName"]
   end
 
+  def winningest_coach(season)
+    coach_game_count = Hash.new(0)
+    coach_wins = Hash.new(0)
```

Comment:
Looks good! We can probably use these default values in the previous methods to cut them down a bit